docs/source/user-guide/tasks/creating-projects.rst

```diff
@@ -0,0 +1,176 @@
+=================
+Creating projects
+=================
+
+In this tutorial, we will walk through how to set up a new Python project in conda
+using an ``environment.yml`` file. This file will help you keep track of your
+dependencies and share your project with others. We cover how to create your
+project, add a simple Python program and update it with new dependencies.
+
+Requirements
+============
+
+To follow along, you will need a working conda installation. Please head
+over to our :doc:`installation guide <../install/index>` for instructions on how
+to get conda installed if you still do not have it.
```

Comment:
```suggestion
to get conda installed if you do not have it.
```
Might just be me, but the "still" makes the tone of the sentence a bit annoyed. Gosh, finally getting around to installing conda, are you? ;)